README.md

```diff
@@ -99,7 +99,7 @@ yarn docker:stop    # don't forget to stop the server when you're done
 If you want to examine or change the database that's used for your local testing, you can run the PostgreSQL interactive command-line client. This will only work after you've run `scripts/resetdb.sh` or `yarn docker:start`.
 
 ```shell
-docker-compose exec postgres psql -U postgres terraware
+docker compose exec postgres psql -U postgres terraware
```

Comment:
Can you update the readme to mention what version of `docker` the user needs to be running?

Reply:
So it looks like support for Docker Compose v2 was added to Docker Desktop back in version `3.4`, so `3.4` or later is all that's required. I'm not sure if we want to list that as the required version or if we want to use a later version? The latest version is `4.32.0`.

Reply:
Went ahead and added a requirements section with Docker Desktop `4.32.0` or later. Happy to change the version or place a note elsewhere if we'd prefer to not add a new section.